src/components/search/SearchCourseCard.jsx

```diff
@@ -45,6 +45,74 @@ const SearchCourseCard = ({ hit, isLoading }) => {
     [JSON.stringify(course)],
   );
 
+  const loadingCard = () => (
+    <Card>
+      <Card.ImageCap
+        as={Skeleton}
+        duration={0}
+      />
+
+      <Card.Header
+        title={
+          <Skeleton duration={0} data-testid="course-title-loading" />
+        }
+      />
+
+      <Card.Section>
+        <Skeleton duration={0} data-testid="partner-name-loading" />
+      </Card.Section>
+
+      <Card.Footer className="bg-white border-0 pt-0 pb-2">
+        <Skeleton duration={0} data-testid="content-type-loading" />
+      </Card.Footer>
+
+    </Card>
+  );
+
+  const searchCourseCard = () => {
+    const primaryPartnerLogo = getPrimaryPartnerLogo(partnerDetails);
+
+    return (
+      <Card
+        isClickable
+      >
+        <Card.ImageCap
+          src={course.cardImageUrl}
+          srcAlt={course.title}
```

Comment:
Let's keep this as an empty string; the background image here is purely decorative.